### DISCUSSION TITLE
WRT3200ACM VLAN setting

### DISCUSSION THREAD

**Discussion by Turro75** [Ysurac/openmptcprouter#2376] [General]
Hello,

I'm not finding a way to adjust the WAN/LAN port, by default the WAN port is set as LAN and the LANs are all set as WAN.
My target is having at least 2 LAN port, Can You kindly help me on getting that?

Thanks

Valerio

**Comment by Ysurac** [comment]
This can be done in Network->Interfaces. You need to change name, disable multipath and change firewall zone.

**Reply by Turro75** [reply]
It worked thanks! easier than a cup of coffee!

I already had deleted the WANx , so it has been enough adding it to the LAN devices in physical setting, save apply and started immediately.
